packages/SwingSet/src/controller.js

```diff
@@ -178,13 +184,40 @@ function makeSESEvaluator() {
     '@agoric/harden': true,
     '@agoric/nat': Nat,
   });
+
+  // Augment the global meter with one that integrates refilling.
+  const refillers = new Set();
```

Comment:
`refillers` should be `endOfCrankHooks`, and it should be passed into `makeSESEvaluator`.

Then, it should be forwarded to `kernelEndowments`.